### DISCUSSION TITLE
How to update new features in Orchard Core?

### DISCUSSION THREAD

**Discussion by TimTracy** [OrchardCMS/OrchardCore#16083] [Q&A]
I am using Orchard Core RC2 I downloaded several months ago.
The new Custom User Settings feature is not appearing under Configuration - Features then scrolling to Settings.
What is the recommended way to get new features updated in my existed copy of Orchard Core.

Thanks,
Tim

**Comment by Skrypt** [comment]
Wether you are using a source code solution or a Nuget Package solution this is different.
If you use source code directly then you can update OC by using Git and merging latest "dev" branch.
If you downloaded source code directly from the repository here as a zip file ; then it might get tricky.

Else, from a Nuget Packages solution all you need to do is look at the this tag here on the main README of this repository 
[![Cloudsmith](https://api-prd.cloudsmith.io/badges/version/orchardcore/preview/nuget/OrchardCore.Application.Cms.Targets/latest/x/?render=true&badge_token=gAAAAABey9hKFD_C-ZIpLvayS3HDsIjIorQluDs53KjIdlxoDz6Ntt1TzvMNJp7a_UWvQbsfN5nS7_0IbxCyqHZsjhmZP6cBkKforo-NqwrH5-E6QCrJ3D8%3D)](https://cloudsmith.io/~orchardcore/repos/preview/packages/detail/nuget/OrchardCore.Application.Cms.Targets/latest/)
And update your project to use this version.

So, give us more details on your solution so that we can help better.

Thanks.

**Comment by hishamco** [comment]
Simpliy you can use the nightly build with cloudsmith packages

**Comment by hishamco** [comment]
Simpliy you can use the nightly build with cloudsmith packages

**Comment by TimTracy** [comment]
I downloaded the Orchardcore.zip RC2 file and extracted the files and using VS2019 16.8.4
I've copied a theme and changing for our needs.  
So I would use from previous comment
   If you use source code directly then you can update OC by using Git and merging latest "dev" branch.
method.

**Comment by TimTracy** [comment]
still open for recommendations.

**Comment by Skrypt** [comment]
- Use `git clone` to get the latest `dev` branch content in a clean new folder.
- Copy / paste your entire OC previous work folder (without any previous /.git folder that could have been sitting there) on top of this new "clean git repo folder".
- Look at the changes/diff you made by opening this new repo folder in a tool like `Gitextensions` or `Fork` to name fews.

If you see changes that are related to OC modules then discard them to keep the latest dev changes unless they are changes that you specifically made. Then, also maybe easier, just copy your custom modules/themes on top of this new clean OC repo folder.

I assume you get the idea.

(optional) Create a private repo on Github ... push it there, have fun 😄

**Comment by mmdqq** [comment]
Can you show the content of the OrchardCore.Media.nuspec file or OrchardCore.Application.Cms.Targets.nuspec
